core/lib/spree/testing_support/factories/product_property_factory.rb

```diff
@@ -7,5 +7,6 @@
   factory :product_property, class: 'Spree::ProductProperty' do
     product
     property
+    deleted_at { nil }
```

Comment:
I don't think this is needed, should be nil by default, right?

Reply:
done